src/WP_CLI/CommandWithUpgrade.php

```diff
@@ -618,5 +624,43 @@ protected function _search( $args, $assoc_args ) {
 	protected function get_formatter( &$assoc_args ) {
 		return new \WP_CLI\Formatter( $assoc_args, $this->obj_fields, $this->item_type );
 	}
+
+	/**
+	 * Get Dropins.
+	 *
+	 * @param array $assoc_args Data passed in from command.
+	 */
+	protected function get_wp_cli_dropin( $assoc_args ) {
+
+		// Get dropins from active plugins.
+		$dropins = get_dropins();
+
+		if ( empty( $dropins ) ) {
+			WP_CLI::error( __( 'No dropins found.' ) );
+		}
+
+		// Get drop-ins that WordPress uses.
+		$wp_dropins = _get_dropins();
+
+		// Get format for result.
+		$format = $assoc_args['format'];
+
+		$items = array();
+
+		foreach ( $dropins as $key => $dropin ) {
+
+			// Get dropin name.
+			$dropin['name'] = $key . ' - ' . $dropin['Name'];
+
+			// Set dropin description.
+			if ( empty( $dropin['Description'] ) ) {
+				$dropin['description'] = $wp_dropins[ $key ][0];
+			}
+
+			$items[] = $dropin;
```

Comment:
Each drop-in should implement all of the fields that a normal plugin array has. If the drop-in doesn't have a corresponding value, you can add an empty string as the value.